src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -21,15 +21,15 @@ export class VehicleTechnicalRecordComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit(): void {
-    this.currentTechRecord = this.technicalRecordService.viewableTechRecord$(this.vehicleTechRecord!, this.ngDestroy$);
+    this.currentTechRecord$ = this.technicalRecordService.viewableTechRecord$(this.vehicleTechRecord!);
```

Comment:
`this.ngDestroy$` is not being used now (apart from in the lifecycle hook of course), should remove it

Reply:
removed